SimpleBlockX4.md

```diff
@@ -0,0 +1,120 @@
+  * Work through important processes on an example that is easy to visualize and understand.
+
+## Procedure Overview
+
+### In Initial CAD program
+_Any CAD engine should work for this process as long as it can export to a
+format importable by SpaceClaim or Cubit/Trelis. The preferred format is ACIS (.sat)._
+  1. Create four solid bricks/cubes with edges of length 100cm that are separated by 50 cm
+    * In Solidworks/Catia you can either make an assembly from 4 cubes or one part
+      with 4 cubes - Either will work, but depending on the ACIS exporter the program
+      may ask you to convert file to part. This is fine, the volumes will remain
+      separate entities in the next steps.
+  2. Export the assembly/part to ACIS format (.sat file extension)
+    * ACIS format 22 is preferred (default for Solidworks)
+    * Note that multiple file types could be used, but there is anecdotal evidence
+      that suggests .sat (ACIS) files work the best for this process. Otherwise formats
+      such as STEP and STL are also able to be imported to Trelis/Cubit and
+      SpaceClaim.
+    * Use save or export feature and select ACIS (.sat) format
+
+### In ANSYS SpaceClaim
+_SpaceClaim is useful for cleaning models that have overlaps, surface conflicts,
+edge contacts, and small gaps. These are further explained in the error blocks example._
+  1. Since the model intrinsically has no conflicts or errors there are no
+  necessary steps in SpaceClaim.
+
+### In Trelis/Cubit
+_Cubit and Trelis are basically the same CAD software with Trelis being the commercial
+version. They are most useful for the meshing of geometries and assigning materials to volumes.
+They have both a GUI and a command line interface and nearly all commands can be run through
+both. For each step the GUI will be explained first, and the command line commands given second.
+"Command Panel" and "Command Line" are necessary for GUI and command line methods respectively.
+To enable these select_ __View__ _and switch "Command Line" and/or "Command Panel" to visible.
+A more general DAGMC based tutorial for Cubit/Trelis basics can be found_ [here](https://svalinn.github.io/DAGMC/usersguide/trelis_basics.html).
+
+  1. Import the .sat file into Trelis/Cubit
+    * Can be imported using the GUI with __File -> Import__. Select the file through file explorer and open. There are further options to select from. For descriptions of these options you can click on the question box on the bottom left, but they will not affect the outcome of this particular import.
+    * In the command line the simple form of the command is "import acis [filepath]" This command can be appended with further commands that can be found in __Help -> Geometry -> Importing -> Importing ACIS files__. These qualifiers correspond with the options given when importing files through the GUI. One important qualifier to include is [attributes_on].

```

Comment:
If you are describing things that people will type in, use single back-ticks to get a useful font.

e.g. `import acis [filepath]`  vs "import acis [filepath]"